ajax/libs/timelinejs/package.json

```diff
@@ -1,28 +1,30 @@
 {
   "name": "timelinejs",
-  "filename": "js/timeline-min.js",
-  "version": "2.36.0",
+  "filename": "js/timeline.min.js",
+  "version": "3.4.7",
   "description": "TimelineJS is an open-source tool that enables you to build visually-rich interactive timelines and is available in 40 languages.",
   "homepage": "https://timeline.knightlab.com/",
   "keywords": [
     "timeline"
   ],
   "autoupdate": {
     "source": "git",
-    "target": "git://github.com/NUKnightLab/TimelineJS.git",
+    "target": "git://github.com/NUKnightLab/TimelineJS3.git",
     "fileMap": [
       {
-        "basePath": "build",
+        "basePath": "",
         "files": [
-          "css/**/*",
-          "js/**/*"
+          "compiled/css/**/*",
+          "compiled/css/*",
+          "compiled/js/**/*",
+          "compiled/js/*"
         ]
       }
     ]
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/NUKnightLab/TimelineJS.git"
+    "url": "https://github.com/NUKnightLab/TimelineJS3.git"
```

Comment:
This one can be combined with the auto-update commit cause it's a part of the version update. Ref: https://github.com/cdnjs/cdnjs/pull/10416#discussion_r102888929